themes/slim.omp.json

```
@@ -33,7 +33,7 @@
           "background": "#2f2f2f",
           "foreground": "#fafafa",
           "style": "diamond",
-          "template": " {{ if ne .Env.POSH_SESSION_DEFAULT_USER .UserName }}<#77f5d6>{{ .UserName }}</><#7a7a7a>\uf1fa</><#2EEFBF>{{ end }}{{ .HostName }}</><#7a7a7a> \ue0b1</>",
+          "template": " {{ if ne .Env.POSH_SESSION_DEFAULT_USER .UserName }}<#77f5d6>{{ .UserName }}</><#7a7a7a>\uf1fa</><#2EEFBF>{{ .HostName }}</><#7a7a7a> \ue0b1</>{{ end }}",
```

The host name should be displayed even for the default user, shouldn't it?

```suggestion
          "template": " {{ if ne .Env.POSH_SESSION_DEFAULT_USER .UserName }}<#77f5d6>{{ .UserName }}</><#7a7a7a>\uf1fa</>{{ end }}<#2EEFBF>{{ .HostName }}</><#7a7a7a> \ue0b1</>",
```

---

@manningli any feedback?

---

Sorry for the slow reply, currently the host name doesn't display for default user in the slimfat theme so I assumed the intent was to remove the segment completely. 

If it's preferred to still keep the host name, moving the `{{ end }}` statement to before the opening color tag would allow the host name to be the correct color and would remove the closing tag artifact.

I don't have a strong preference either way, but after going through a few new machines at work recently, having to modify the theme to remove the `<\>` from the slimfat theme each time felt like some change is needed. Regardless of which direction is preferred, I'll be available to make the update quicker now.

---

Just noticed I missed @lewis-yeung suggested change in the comment mirroring what I just said... I'll commit that so host name remains and make the same change to slimfat for consistency.